File: backend-arso/opiniones/src/main/java/modelo/Valoracion.java
package modelo;

import java.io.Serializable;

import org.bson.BsonType;
import org.bson.codecs.pojo.annotations.BsonId;
import org.bson.codecs.pojo.annotations.BsonProperty;
import org.bson.codecs.pojo.annotations.BsonRepresentation;

import utils.Identificable;

public class Valoracion implements Identificable, Serializable {

	private static final long serialVersionUID = 1L;

	@BsonId
	@BsonRepresentation(BsonType.OBJECT_ID)
	private String id;
	@BsonProperty
	private String correo;
	private int calificacion;
	private String comentario;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getCorreo() {
		return correo;
	}

	public void setCorreo(String correo) {
		this.correo = correo;
	}

	public int getCalificacion() {
		return calificacion;
	}

	public void setCalificacion(int calificacion) {
		this.calificacion = calificacion;
	}

	public String getComentario() {
		return comentario;
	}

	public void setComentario(String comentario) {
		this.comentario = comentario;
	}

	@Override
	public String toString() {
		return "Valoracion [id=" + id + ", correo=" + correo + ", calificacion=" + calificacion + ", comentario="
				+ comentario + "]";
	}

}

File: src/js/beans/Valoracion.js
class Valoracion {

    // Propiedades
    Id;
    Correo;
    Calificacion;
    Comentario;
    IdUser;
  
    // Metodos
    fromJson(json){
      this.Id = json.id;
      this.Correo = json.correo;
      this.Calificacion = json.calificacion;  
      this.Comentario = json.comentario;  
      this.IdUser = json.idUser;  
    }
    
    print(){
      console.log("Valoracion: { Id: " + this.Id +", Correo: " + this.Correo +  ", Calificacion: " 
      + this.Calificacion + ", Comentario: " + this.Comentario + ", IdUser: " + this.IdUser + " }");
    }
  }

exports.Valoracion = Valoracion;